xray/plot/plot.py

```diff
@@ -337,7 +411,8 @@ def _plot2d(plotfunc):
     @functools.wraps(plotfunc)
     def newplotfunc(darray, ax=None, xincrease=None, yincrease=None,
                     add_colorbar=True, vmin=None, vmax=None, cmap=None,
-                    center=None, robust=False, extend=None, **kwargs):
+                    center=None, robust=False, extend=None, levels=None,
```

Comment:
We'll want to update the signature for `plotmethod` also to include this. My fault on having to maintain two signatures like this- have been thinking if there's a better way.